tests/test_nanoevents_edm4hep.py

```diff
@@ -19,14 +19,14 @@ def _events(**kwargs):
 @pytest.fixture(scope="module")
 def eager_events():
     return _events(
-        mode="eager", iteritems_options={"filter_name": lambda x: "PARAMETERS" not in x}
+        mode="eager", iteritems_options={"filter_name": "/^(?!.*(PARAMETERS|_.*Map))/"}
```

Comment:
I think that this is a very specific case. Perhaps, finding a more generic solution would be better?